pkg/tpm/tpm.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2015 CoreOS, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build tpm
+
+package tpm
+
+import (
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/coreos/go-tspi/tpmclient"
+)
+
+// Extend extends the TPM log with the provided string. Returns any error.
+func Extend(desription string) error {
```

Comment:
Ouch! Damn, I must have messed up setting the flag when testing the build (I'd just moved the code with no functional changes). @jonboulle The dependencies are in Ubuntu - we just need libtspi-dev installed.